setup.py

```diff
@@ -62,8 +62,8 @@
     packages=find_packages(exclude=['tests*', 'examples']),
 
     install_requires=[
-        'base58==0.2.2',
-        'PyNaCl==1.0.1',
+        'base58>=0.2.2',
+        'PyNaCl>=1.0.1',
```

Comment:
This will install any version bigger than `0.2`. If `base58` and `PyNaCl` are using semantic versioning this selector might install some backward incompatible package. Can we use the `~` operator or something similar?